### PR TITLE
feat(local-node): Added support for validium and custom base token

### PR DESCRIPTION
**Pull request by mm-zk**
## What ❔

* Added support for validium and custom base token for local node.

### CODE REVIEW

File: docker/local-node/entrypoint.sh [hunk 1]
```diff
@@ -146,8 +146,22 @@ else
     zk contract initialize-validator
   fi
 
-  ## init hyperchain
-  zk contract register-hyperchain
+
+  if [ -z "$CUSTOM_BASE_TOKEN" ]; then
+    echo "Starting chain with ETH"
```
Comment (shahar4) [inline docker/local-node/entrypoint.sh:151]: nit: might be a confusing log if it doesn't specify it's a gas token (also below)

"Starting chain with ETH as a gas token" instead?

File: docker/local-node/entrypoint.sh [hunk 2]
```diff
@@ -146,8 +146,22 @@ else
     zk contract initialize-validator
   fi
 
-  ## init hyperchain
-  zk contract register-hyperchain
```
Comment (shahar4) [inline docker/local-node/entrypoint.sh:150]: Is there a chance something else is relying on this? The new CLI perhaps?

File: docker/local-node/entrypoint.sh (hunk 1 above)
Comment (mm-zk) [reply docker/local-node/entrypoint.sh:151]: done.

File: docker/local-node/entrypoint.sh (hunk 2 above)
Comment (mm-zk) [reply docker/local-node/entrypoint.sh:150]: it is still executed - in line 155.

'the -z' option means that a given env variable is not set.

**Review by shahar4 — APPROVED** [review]

**Pull request merged**